hi-ml-multimodal/README.md

```diff
@@ -1,11 +1,71 @@
-# Multimodal
+# HI-ML Multimodal Toolbox
 
-## Developer setup
+This toolbox provides models for working with multi-modal health data.
+The code is available on [GitHub][1] and [Hugging Face 🤗][6].
 
-Make file commands:
+## Getting started
 
-* `make pip_local` to install the `hi-ml-multimodal` package in editable mode. This must happen before running tests.
-* `make build` to build the package
-* `make mypy` to run `mypy`
-* `make check` to run `flake8`, `mypy` and `pyright`
-* `make clean` to clean up all temporary files and folders
+The best way to get started is by running the [phrase grounding notebook][2].
+All the dependencies will be installed upon execution, so Python 3.7 and [Jupyter][3] are the only requirements to get started.
+
+The notebook can also be run on [Binder][4], without the need to download any code or install any libraries:
+
+[![Binder](https://mybinder.org/badge_logo.svg)][4]
+
+## Installation
+
+The latest version can be installed using `pip`:
+
+```console
+pip install "git+https://github.com/microsoft/hi-ml.git#subdirectory=hi-ml-multimodal"
+```
+
+### Development
+
+For development, it is recommended to clone the repository and set up the environment using [`conda`][5]:
+
+```console
+git clone https://github.com/microsoft/hi-ml.git
+cd hi-ml-multimodal
+make env
+```
+
+This will create a `conda` environment named `multimodal` and install all the dependencies to run and test the package.
+
+You can visit the [API documentation][9] for a deeper understanding of our tools.
+
+## Hugging Face 🤗
+
+More information about our models and trained weights can be found on [Hugging Face 🤗][6].
```

Comment:
Maybe we could add that our trained models and weights are hosted on Hugging Face, and that this repo provides examples on how to use these models?

I don't think I got the wording quite right. Basically, I think it could be useful to let people know what the purpose of this repo is, considering the models are already on Hugging Face.

Reply:
That's a good point. I wasn't sure what to write in this section. I've rephrased in c488898, hopefully it's a bit better.